pkg/frontend/authenticate.go

```diff
@@ -3279,88 +3285,24 @@ func doCreateStage(ctx context.Context, ses *Session, cs *tree.CreateStage) (err
 func doCheckFilePath(ctx context.Context, ses *Session, ep *tree.ExportParam) (err error) {
 	//var err error
 	var filePath string
-	var sql string
-	var erArray []ExecResult
-	var stageName string
-	var stageStatus string
-	var url string
 	if ep == nil {
 		return err
 	}
 
-	bh := ses.GetBackgroundExec(ctx)
```

Comment:
enable/disable in stage status is invalid.  Check enable/disable should be removed.  Path expansion from stage URL to final path is done in sql/plan/function/stage_util.go